ncovstats/src/country.js

```diff
@@ -35,17 +35,22 @@ async function _command(params, commandText, secrets = {}) {
   const result = {};
 
   const html = cheerio.load(response.data);
-  const country = toTitleCase(params.country_name); 
+  let country = toTitleCase(params.country_name);
+  country = abbrExpand(country);
   const countryStat = html(`td:contains(${country})`);
-  if(countryStat.length)
-  {
+  let msg;
+  if (countryStat.length) {
     result.cases = countryStat.next().text();
     result.deaths = countryStat.next().next().text();
+    msg = `CoronaVirus :mask: Stats in *${country}* ${flagger(country)} :\n *Cases:-* ${result.cases} \n *Deaths:-* ${result.deaths}`;
```

Comment:
If we use the same case names as `abbrExpand` in `flagger`, we can get rid of another assignment `country = abbrExpand(country)` and just use `${abbrExpand(country)}` instead. 

And also, what do you think about using the name `getFlag` instead of `flagger`?

Reply:
here `country = abbrExpand(country)` is assigned once and used twice, if we remove this assignment we will need to call `abbrExpand` twice. Also `abbrExpand` is required because of the fact I mentioned in above comment. 

> what do you think about using the name getFlag instead of flagger?
That makes perfect sense, I'll update the PR

Thanks for the feedback!

Reply:
Yes, that's the result of me not understanding the user input formatting clearly. Thanks for the explanation.

Reply:
that's fine, we learn together! Thanks.